lib/WebpackError.js

```diff
@@ -4,8 +4,81 @@
 */
 "use strict";
 
+const createHash = require("./util/createHash");
+
+const hash = Symbol();
+const meta = Symbol();
```

Comment:
Could you name the symbols?

Reply:
For what purpose? There's no best practice outlining that should be done, and these are not meant to be used outside of the class.

Reply:
This is for debugging purpose. Here below, you can easy identify what `b` is whereas `a` is not.

![symbols](https://user-images.githubusercontent.com/168498/37734900-ca07c4c4-2d4c-11e8-9b91-c16ffcd59a4f.png)

Reply:
Fair enough. I'll update this.